### ISSUE TITLE
label

### ISSUE THREAD

**Issue by 99sun99** [Jack-Etheredge/Brain-Tumor-Segmentation-3D-UNet-CNN#6]
do you put the classification label and the mask together into the  model?  I can not understand how you do it. do you try to write a one-hot code?

**Comment by Jack-Etheredge** [comment]
Yeah. Hopefully the refactored code is a lot easier to interpret. I created separate layers for each tumor region, so they're binarized. I guess you can think of that as one-hot encoding. If you wanted to submit the output, you'd have to reverse that process into a single image and then also reverse the cropping. Reversing the cropping would be the much more challenging of the two, honestly.
